frontend/src/components/Files/ObjectDeleteModal.tsx

```diff
@@ -23,24 +23,19 @@ export const ObjectDeleteModal = ({
 }) => {
   const isOpen = cid !== null;
   const [metadata, setMetadata] = useState<UploadedObjectMetadata | null>(null);
-  const session = useSession();
 
   useGetMetadataByHeadCidQuery({
     variables: {
       headCid: cid ?? '',
     },
-    skip: !cid || !session.data?.accessToken,
+    skip: !cid,
     onCompleted: (data) => {
       setMetadata(mapObjectInformationFromQueryResult(data));
     },
     onError: (error) => {
       console.error('error', error);
     },
-    context: {
-      headers: {
-        Authorization: `Bearer ${session.data?.accessToken}`,
-      },
-    },
+    client: gqlClient,
```

Comment:
One other approach will be to create a ApolloProvider and use it in layout to wrap your children's in the provider, so you won't need to pass in the client on each query

Reply:
Yeah, that's the best in this, I already had the ApolloProvider setup.

Reply:
Thanks for the comment btw!